packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -460,6 +475,10 @@ class ScrollableBase extends Component {
 		}
 	}
 
+	onScrollToTopButtonClick = () => {
+		this.uiRef.current.start({targetX: 0, targetY: 0});
```

Comment:
Expect we'll want to focus the first item along with this which I think means we'll want to use `scrollTo` rather than this (more) private method. @germboy or @ybsung can likely help on this.

Reply:
As @ryanjduffy mentioned, calling `scrollTo` would be more safer than calling `start` directly because the states in the `VirtualList` or the `Scrollable` could be conflict while swapping items. In this case, calling `start` would be deferred internally in the `scrollTo` function.

In addition, we usually check several points before jumping to another index.
In the `onAcceleratedKeyDown` of the `VirtualList`,
1) if the next item (the 0 index item in this case) is in a current page, we just focus it.
2) if the next item is out of a current page, we just blur the current focused item and call the `scrollTo` with an index (the 0 in this case).